whois-nrtm4-client/src/main/java/net/ripe/db/nrtm4/client/dao/Nrtm4ClientInfoRepository.java

```diff
@@ -64,7 +68,7 @@ SELECT id, source, MAX(version), session_id, type, hostname, created
     @Nullable
     public NrtmClientVersionInfo getNrtmLastVersionInfoForDeltasPerSource(final String source){
         try {
-            return jdbcMasterTemplate.queryForObject("""
+            return jdbcSlaveTemplate.queryForObject("""
```

Comment:
Why are you using the *slave* datasource here when elsewhere you're using the master datasource during import ? Is it possible that the slave datasource will be out-of-sync ?

Reply:
yes, I will use master

Reply:
Done